apps/experiments/views/experiment.py

```diff
@@ -415,8 +415,7 @@ def _get_voice_provider_alpine_context(request):
 def delete_experiment(request, team_slug: str, pk: int):
     safety_layer = get_object_or_404(Experiment, id=pk, team=request.team)
     safety_layer.delete()
-    messages.success(request, "Experiment Deleted")
```

Comment:
This message shows up very briefly when we use the `HX-Redirect` approach. I _think_ it might be because htmx gets the response and redirects the page again to the url in the HX-Redirect header. This would explain why the toast shows up and disappears again.